service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/CircuitBreakersConverter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.client;
```

Comment:
Does it need to be a public API? I don't think that users will need to use it directly. I think we can put it in an internal package.